app/templates/index.html

```diff
@@ -90,18 +86,17 @@ <h1>'Allo, 'Allo!</h1>
       r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
       ga('create','UA-XXXXX-X');ga('send','pageview');
     </script>
-
-    <!-- build:js scripts/vendor.js -->
-    <!-- bower:js -->
-    <!-- endbower -->
-    <!-- endbuild -->
+    <!-- build:js scripts/vendor.js -->	
+    <% if (includeModernizr) { %>
+    <script src="scripts/modernizr.js"></script>
```

Comment:
Please keep Modernizr in `<head>`, otherwise there might be a flash of outdated content, because Modernizr can affect CSS too, which means that the page will first load the styles **before** Modernizr was loaded. You can surround it in build comments with the name `scripts/vendor/modernizr.js` like it was before.

Reply:
I restored it 👍